ui/insight/src/view.ts

```diff
@@ -80,18 +61,8 @@ module.exports = function (ctrl) {
         ctrl.vm.panel === 'preset' ? presets(ctrl) : null,
         help(ctrl),
       ]),
-      m('header', [
-        axis(ctrl),
-        m(
-          'h2',
-          {
-            class: 'text',
-            'data-icon': '',
-          },
-          'Chess Insights'
-        ),
-      ]),
-      m('div.insight__main.box', renderMain(ctrl)),
-    ]
+      h('header', [axis(ctrl), h('h2.text', { attrs: { 'data-icon': '' } }, 'Chess Insights')]),
+      thunk('div.insight__main.box', renderMain, [ctrl, cacheKey(ctrl)]),
```

Comment:
We get to use snabbdom thunks instead of mithril's low-level `{ subtree: 'retain' }` here.